boards/st/nucleo_h745zi_q/nucleo_h745zi_q_stm32h745xx_m7.dts

```diff
@@ -144,3 +144,13 @@ zephyr_udc0: &usbotg_fs {
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+&fdcan1 {
+	pinctrl-0 = <&fdcan1_rx_pb8 &fdcan1_tx_pb9>;
+	// HSE will be used by default. Uncomment below to enable APB1.2 120MHz clock
+	// clocks = <&rcc STM32_CLOCK_BUS_APB1_2 0x00000100>,
+	//	    <&rcc STM32_SRC_PLL1_Q FDCAN_SEL(1)>;
+	pinctrl-names = "default";
+	// pins are confilcting with i2c1 - disable i2c1 first to enable
```

Comment:
This is fine to me, but please use C style comments.

Reply:
can you please elaborate ; 
the board schematics and the userManual (Table 17. ) gives on connector CN9
CAN_RX PD0 CAN_1 
CAN_TX PD1 CAN_1 
So changing this configuration will not only conflict with I2C 1 pinning but it will also mismatch the nucleo board specification.

Reply:
Thank you both for your valuable comments. 
@erwango I will change to C style comments.

@FRASTM  You're right. I've choosen fdcan1 pins that way, cause I am using [waveshare's rs485-can shield](https://www.waveshare.com/rs485-can-shield.htm), since it it only shield (i know), which can be plugged to almost any nucle board. Thus very handy. 
But it is true - it will make more sense in the current context to move to pins set mentioned by you, s.t. can can be enabled by default and tests will run. Afterwards the specific projects may alter pins for own need e.g. waveshare shield in my case.